core/src/foundations/ionic.root.scss

```diff
@@ -0,0 +1,120 @@
+/**
+ * This is an auto-generated file, please do not change it directly.
+ * Ionic Design System
+ */
+
+:root {
+  --ionic-color-primary-10: #f7faff;
```

Comment:
Yes, we did! 

In summary, as we are introducing a new Design System, we wanted to avoid confusion and conflicts with the current one. 

As the new theme is also called Ionic, we thought the `--ionic` prefix makes total sense for these new tokens. 

This decision is also present on the [Decision Log](https://outsystemsrd.atlassian.net/wiki/spaces/RDMBVS/pages/4057366529/Native+Look+Feel+-+Decisions+Log) page 👍

Reply:
sounds good, thanks!